Solutions/SAP BTP/Analytic Rules/BTP - Failed access attempts across multiple BAS subaccounts.yaml

```diff
@@ -0,0 +1,44 @@
+id: 74b243a6-3046-48aa-8b03-e43b3c529cc1
+kind: Scheduled
+name: BTP - Failed access attempts across multiple BAS subaccounts
+description: Identifies failed Business Application Studio access attempts over a
+  predefined number of subaccounts.
+severity: Medium
+status: Available
+requiredDataConnectors:
+  - connectorId: SAPBTPAuditLog
+    dataTypes:
+      - SAPBTPAuditLog_CL
+queryFrequency: 15m
+queryPeriod: 1d
```

Comment:
QueryFrequency and QueryPeriod should be in sync. If we run the query for every 15mins, entire day of data would be evaluated 24*4 times. This will lead to duplicate results.

Reply:
Synced values. They were originally configured with Incident grouping but this is not supported for Templates.